_data/subnav.yml

```diff
@@ -85,6 +85,8 @@ subnav:
         url: source/items/stash
       - page: Supply Camp and Supply Ship
         url: source/items/supplycampandship
+      - page: Tag Tool and Tag Anchor
```

Comment:
nitpick, but maybe tag anchor _block_ here?

Reply:
I thought about it, but it's in the "Items/Blocks" menu so that seemed redundant.  And it already wraps a bit so conciseness seems better.